node/src/exchange/client.rs

```diff
@@ -34,7 +61,7 @@ impl ExchangeClientHandler {
     #[instrument(level = "trace", skip(self, req_resp, respond_to))]
     pub(super) fn on_send_request(
         &mut self,
-        req_resp: &mut ReqRespBehaviour,
+        req_resp: &mut R,
```

Comment:
please change all args `req_resp` to just `req`

Reply:
I don't like the name clash with request tho :thinking:

Reply:
maybe to `sender` then?

Reply:
I'm again having feelings with this `Request` trait name, it's confusing that it's used just next to `Request` which is just a data structure. If you don't mind I find `SendReq` or `ReqSender` much less confusing

Reply:
@fl0rek wdyt?

Reply:
maybe just `Sender`?

Reply:
changed to `Sender`